be2-scala/src/test/scala/ch/epfl/pop/authentication/WebSocketResponseHandlerSuite.scala

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.pop.authentication
+
+import akka.http.scaladsl.testkit.{ScalatestRouteTest, WSProbe}
+import ch.epfl.pop.config.RuntimeEnvironment
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+class WebSocketResponseHandlerSuite extends AnyFunSuite with Matchers with ScalatestRouteTest {
```

Comment:
You should include the Testkit with the name of the class of test for more clarity (as it is done in the other files) :)